dll/win32/mshtml/olecmd.c

```diff
@@ -241,6 +241,13 @@ static HRESULT exec_print(HTMLDocument *This, DWORD nCmdexecopt, VARIANT *pvaIn,
         return S_OK;
     }
 
+#ifdef __REACTOS__
+    // returning here fixes CORE-16884. Maybe use this until printing works.
+    ERR("Sorry, ReactOS does not support printing yet.\n");
+    return S_OK;
+
+    // For now if this is executed we get internal corruptions for unknown reasons. 
```

Comment:
This line does not seem wanted in code. The reference to CORE-16884 should be enough.